### PR TITLE
Update list.md

### PR DESCRIPTION
**Pull request by pi-core-team**

### CODE REVIEW

**Review by CryptoGing — APPROVED** [review]

File: list.md [hunk 1]
```diff
@@ -68,5 +68,5 @@ To add your project to the list follow the instructions [here](/README.md)
 | DCert | Application for issuing and maintaining certificates on blockchain | Consumer Utility, Education, Business Tool | JavaScript/Html, Node.js/Express, MongoDB | [dcert](https://github.com/pi-apps/dcert) | pi://brainstorm.pi/project/63df8abe8fa69420e171e704 | pi://dcert-40a5b.web.app |  |
```
Comment (wasi4440) [inline list.md:68]: #80

File: list.md [hunk 2]
```diff
@@ -68,5 +68,5 @@ To add your project to the list follow the instructions [here](/README.md)
...
 | store on Pi | A platform with many physical stores, users can purchase goods on the platform and get QR codes and go to physical stores for consumption, and use PI payment.| Consumer utilities | SpringCloud/java, JS/Vue,Mysql,Mongodb,Kubernetes ,docker etc. | [store-on-pi](https://github.com/pi-apps/store-on-pi) | pi://brainstorm.pi/project/63c641ea731cc60885ea2f7f | (When available) | (When available) |
 | QRcodePay | A Tool for Pioneers to make payment easily by scanning QRcode | Ecosystem Tools | NextJS/NodeJS/Firebase/Javascript | [qrcodepay](https://github.com/pi-apps/qrcodepay) | pi://brainstorm.pi/project/63f8c8069326fc9c895bedba |  [pi://pi-qrcode.vercel.app/](pi://pi-qrcode.vercel.app/) | (When available) |
-| Voice Of Pi | Voice Of Pi is a Social Finance platform based on the world’s most accessible blockchain: Pi Network. | Social Media | JavaScript/Html/Bootstrap, Express.js | [Voice Of Pi](https://github.com/PiVoice/PiVoice) | pi://brainstorm.pi/project/63bce50944c7690233094b2d | https://pivoice.app | (When available) |
```
Comment (wasi4440) [inline list.md:71]: #80 #81 #79

**Review by haticecan37 — COMMENTED** [review]
list.md

File: list.md (hunk 2 above)
Comment (haticecan37) [reply list.md:71]: https://github.com/pi-apps/PiOS/pull/64/files

**Review by Kamicio1234 — APPROVED** [review]

**Review by Kamicio1234 — APPROVED** [review]

**Pull request merged**